src/components/LemmaInput.vue

```diff
@@ -39,14 +39,23 @@ export default {
 </script>
 
 <template>
-	<text-input
-		class="mw-wbl-snl-lemma-input"
-		:label="messages.get( 'wikibaselexeme-newlexeme-lemma' )"
-		:placeholder="messages.get( 'wikibaselexeme-newlexeme-lemma-placeholder' )"
-		name="lemma"
-		required
-		:error="buildError( error )"
-		:value="modelValue"
-		@input="$emit( 'update:modelValue', $event )"
-	/>
+	<div class="mw-wbl-snl-lemma-input">
```

Comment:
I like where you're coming from here, this enables us to be more flexible in case particular inputs have different spacing requirements.

But I'm not sure if the extra div is worth it. Especially since we could utilize flex and grid to manipulate the layout on the outer layer of the form, which also might be more appropriate in terms of separation of concerns.